## research-project-2021 | Final year undergraduate research project
# Co-occupying Swi/Snf and Ssn6-Tup1 chromatin remodelling activities are required for full nucleosome free region accessibility at Ssn6 promoter targets
### William Whittick | Kent Lab at Cardiff University School of Biosciences | 2021


<img width="227" alt="image" src="https://user-images.githubusercontent.com/82673987/162563582-04193430-20f5-4569-88c4-274c999d6c68.png">

ABSTRACT

**Background**: By altering nucleosome positioning and structure, eukaryote co-activator and co- repressor remodelling factors change the structure of chromatin to positively and negatively regulate gene expression respectively. Swi/Snf and Ssn6-Tup1 complexes are such factors whose distinct binding and co-occupancy has been identified as a requirement for repressive chromatin structure remodelling at promoter regions and upstream domains. Micrococcal nuclease (MNase) directed chromatin particle spectrum analyses offer an opportunity to map nucleosome landscapes throughout the Saccharomyces cerevisiae genome at a range of specified features.

**Materials and Methods**: MNase digested DNA was sequenced from Ssn6 knockout (Ssn6∆), Snf2 knockout (Snf2∆) and Snf2/Ssn6 double knockout (Snf2∆Ssn6∆) mutant S. cerevisiae strains providing 150bp nucleosomal paired reads for processing by a suite of Perl executable scripts. Chromosomal nucleosome distributions were visualised in the Integrated Genome Browser by plotting nucleosome frequency against genomic positions. Typical nucleosome occupancies were determined for a range of genomic features by plotting normalised cumulative nucleosome frequency distributions and clustered into common occupancy patterns using Cluster 3.0.

**Results**: Insert size distribution chromatin particle size classes reflect input MNase-digested DNA size spectra for WT and Ssn6∆ bio-replicates and, among samples, bio-replicates correlate very strongly. A Ty1 retrotransposon duplication event (chrIV:1 101 332-1 206 996) was described by apparent abnormal nucleosome frequency that results in no change to typical retrotransposon gene nucleosome occupancy. Nucleosome occupancy data provide evidence suggesting regulatory interactions between Ty1 duplication and tRNA gene expression. Swi/Snf and Ssn6-Tup1 co-occupancy leads to the most accessible NFR formation at Ssn6 binding promoter regions and ChIP-exo Ssn6 binding regions. This finding was reflected in nucleosome positioning at hexose transporter encoding gene HXT2 and HXT7 promoter regions.

**Conclusions**: Further studies on Ssn6-Tup1 chromatin remodelling activity should include sequence tagged Ty1 elements to identify Ty1 insertion sites and consider this against tRNA gene nucleosome occupancy. Co-occupying Swi/Snf and Ssn6-Tup1 chromatin structure remodelling is transcriptionally activating by increased NFR accessibility at Ssn6 binding promoter regions. Despite evident Ssn6-Tup1 repression, HXT2 and HXT7 loci experience these co-occupying effects demonstrating a remodelling opposing interplay.


##### Wild-type budding yeast reference strain BY4742 (Saccharomyces cerevisiae, genotype: MATa; his3∆1; leu2∆0; lys2∆0; ura3∆0) was obtained from EUROSCARF

##### _SSN6_ and _SNF2_ Mutant strain deletions were made using standard gene replacement methods (Wach et al. 1994) by Prof. Alastair Fleming and Dr. Mohamed Alhussain at The Moyne Institute, trinity College Dublin

##### Dr. Mohamed Alhussain carried out yeast culturing and MNase chromatin digestion using methods described by Kent et al. 2011 and Kent and Mellor (1995)

##### DNA sequencing libraries were prepared using the NEXTFLEX Rapid kit (Bioone/Perkin Elmer) and sequenced on an Illumina Nextseq500 in 50bp paired end mode at the Cardiff School of Biosciences Genomics Research Hub
